consensus/types/src/execution_bid.rs

```diff
@@ -0,0 +1,47 @@
+use crate::{test_utils::TestRandom, *};
+use derivative::Derivative;
+use serde::{Deserialize, Serialize};
+use ssz_derive::{Decode, Encode};
+use test_random_derive::TestRandom;
+use tree_hash_derive::TreeHash;
+
+#[derive(
+    Default,
+    Debug,
+    Clone,
+    Serialize,
+    Encode,
+    Decode,
+    Deserialize,
+    TreeHash,
+    Derivative,
+    TestRandom,
+)]
+#[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
+#[derivative(PartialEq, Hash)]
+#[context_deserialize(ForkName)]
+// This is what Potuz' spec calls an `ExecutionPayload` even though it's clearly a bid.
```

Comment:
kept this as `ExecutionBid` per our convo, in lieu of `ExecutionPayloadHeader` per [spec](https://ethereum.github.io/consensus-specs/specs/_features/eip7732/beacon-chain/#executionpayloadheader)